crates/goose-mcp/src/computercontroller/mod.rs

```diff
@@ -89,29 +92,29 @@ impl ComputerControllerRouter {
         let computer_control_tool = Tool::new(
             "computer_control",
             indoc! {r#"
-                Control the computer using AppleScript (macOS only). Automate applications and system features.
-
-                Key capabilities:
-                - Control Applications: Launch, quit, manage apps (Mail, Safari, iTunes, etc)
-                    - Interact with app-specific feature: (e.g, edit documents, process photos)
-                    - Perform tasks in third-party apps that support AppleScript
-                - UI Automation: Simulate user interactions like, clicking buttons, select menus, type text, filling out forms
-                - System Control: Manage settings (volume, brightness, wifi), shutdown/restart, monitor events
-                - Web & Email: Open URLs, web automation, send/organize emails, handle attachments
-                - Media: Manage music libraries, photo collections, playlists
-                - File Operations: Organize files/folders
-                - Integration: Calendar, reminders, messages
-                - Data: Interact with spreadsheets and documents
+                Control the computer using system automation. Features available:
```

Comment:
would this be better to have conditional logic based on the os detection? then it won;t be polluting the prompt with un-necessary instructions (and can be consistent with how it was with windows having its own set). As it is - hard to test

Reply:
will separate